website/docs/r/config_remediation_configuration.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "aws"
+page_title: "AWS: aws_config_remediation_configuration"
+sidebar_current: "docs-aws-resource-config-remediation-configuration"
+description: |-
+  Provides an AWS Config Remediation Configuration.
+---
+
+# Resource: aws_config_remediation_configuration
+
+Provides an AWS Config Remediation Configuration.
+
+~> **Note:** Config Remediation Configuration requires an existing [Config Rule](/docs/providers/aws/r/config_config_rule.html) to be present. Use of `depends_on` is recommended (as shown below) to avoid race conditions.
+
+## Example Usage
+
+AWS managed rules can be used by setting the source owner to `AWS` and the source identifier to the name of the managed rule. More information about AWS managed rules can be found in the [AWS Config Developer Guide](https://docs.aws.amazon.com/config/latest/developerguide/evaluate-config_use-managed-rules.html).
+
+```hcl
```

Comment:
For examples in the documentation, it's better for them to be short so that the essential parts are more obvious. In this case, we'd be ok to remove the IAM resources, or possibly leave the definition but remove the attributes. It does have the trade-off that these long examples aren't executable as-is.